packages/api/cms-api/src/auth/guards/comet.guard.ts

```diff
@@ -2,10 +2,12 @@ import { CanActivate, ExecutionContext, Inject, Injectable, UnauthorizedExceptio
 import { Reflector } from "@nestjs/core";
 import { GqlContextType, GqlExecutionContext } from "@nestjs/graphql";
 import { Request } from "express";
+import { SystemUser } from "src/user-permissions/user-permissions.types";
```

Comment:
I thought absolute imports don't work in packages... has this changed? I'd still use a relative import here to be consistent.